week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,10 +7,14 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  return price *1.3*1.01;
+}
+function convertToBRL(price) {
+  return convertToUSD(price)*3.73;
```

Comment:
Putting the extra 1% in the USD conversion and then sending all other conversions through USD is an interesting solution to the problem. I like your thinking!

The question I would like to ask you is this: if another developer came to this code, would they understand why `convertToBRL` must call `convertToUSD` and not do the conversion straight away? Is it clear from reading the code that all foreign currency conversions should have an extra 1% applied?

I'd like you to think about another way you could write this that would make that part easier to understand. Maybe it's difficult, but it is good for you to think about. 😄